cypress/integration/projectSettings/project_settings.ts

```diff
@@ -290,628 +308,589 @@ describe("Repo Settings", { testIsolation: false }, () => {
       cy.contains("button", "Variant/Task").click();
       cy.dataCy("variant-regex-input").type(".*");
       cy.dataCy("task-regex-input").type(".*");
-      cy.dataCy("github-trigger-alias-checkbox").check({ force: true });
-
+      cy.getInputByLabel("Add to GitHub Trigger Alias")
+        .as("triggerAliasCheckbox")
+        .should("not.be.checked");
+      cy.contains("label", "Add to GitHub Trigger Alias").click();
```

Comment:
Use getInputBylabel here or the alias you created above.

Reply:
I do agree that Leafygreens "fake checkbox" implementation isn't ideal. I think I would still prefer that we use the same pattern we use elsewhere in the code base. It is more readable and clear what we are trying to check and assert on the same checkbox element.  
```suggestion
            cy.get("@triggerAliasCheckbox").check({ force: true });
```